src/datasets/tasks/question_answering.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from typing import Dict
+
+from ..features import Features, Sequence, Value
+from .base import TaskTemplate
+
+
+@dataclass(frozen=True)
+class QuestionAnswering(TaskTemplate):
```

Comment:
`QuestionAnsweringExtractive` ?

Reply:
thanks for the suggestion! i'll keep this unchanged for now since it will soon be overhauled by #2371